hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/VolumeManagerImpl.java

```diff
@@ -453,23 +449,11 @@ public boolean checkVolumeAccess(String volume, OzoneAclInfo userAcl)
   @Override
   public List<OmVolumeArgs> listVolumes(String userName,
       String prefix, String startKey, int maxKeys) throws IOException {
-    metadataManager.getLock().acquireLock(USER_LOCK, userName);
+    metadataManager.getLock().acquireWriteLock(USER_LOCK, userName);
```

Comment:
Do we really need a write lock for list volume operation?

Reply:
This is merely cleaning up the deprecated call.

Reply:
I could either leave it alone, or change it to `acquireReadLock`. The latter should be fine though.

Reply:
I think we should change it to acquireReadLock.

Reply:
done: c9a92918ef0363ee711222688bf8d851e6321418